.github/workflows/rust.yml

```diff
@@ -415,3 +415,46 @@ jobs:
           export RUSTFLAGS="-C debuginfo=0"
           cd rust/arrow
           cargo build --target wasm32-unknown-unknown
+
+  # test the projects can build without default features
+  default-build:
+    name: AMD64 Debian 10 Rust ${{ matrix.rust }} test no defaults
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64]
+        rust: [nightly-2020-11-24]
```

Comment:
Should we change this to `stable`(or have both), since this is the target version?

Reply:
yes, that is a good idea. I will make the change

Reply:
in https://github.com/apache/arrow/pull/9904